ably-ios/ARTAuth.h

```diff
@@ -30,7 +30,7 @@ ART_ASSUME_NONNULL_BEGIN
 @property (nonatomic, weak) ARTLog *logger;
 
 @property (art_nullable, readonly, getter=getClientId) NSString *clientId;
-@property (art_nullable, nonatomic, readonly, strong) ARTAuthTokenDetails *tokenDetails;
+@property (art_nullable, nonatomic, readwrite, strong) ARTAuthTokenDetails *tokenDetails;
```

Comment:
It is a mistake. I removed the `setTokenDetails` and I change it to `readwrite` just to compile the code successfully and continue the remaining work of this spec.